### ISSUE TITLE
Question about hyperparameters of DBScan

### ISSUE THREAD

**Issue by haiphamcse** [astra-vision/PaSCo#1]
Hi there, again love your work! I wanted to ask about the progress of generating panoptic pseudo ground-truth from SemKITTI. I tried to use DBScan with your config (a distance of ϵ = 1 and groups with MinPts = 8) and generate AABB box for each cluster. However I do notice sometimes the clustering is not good for cars that are close to each other. Have you encountered this problem? Thank you.
![image](https://github.com/astra-vision/PaSCo/assets/76871780/29445b8b-57ea-412e-9e71-12fef7ff9377)

**Comment by anhquancao** [comment]
Hello @haiphamcse,
Thank you for your interest! We've decided to use this ground-truth as is, despite its imperfections. This is still fair, as all methods are re-trained and evaluated under the same conditions. Also, it's not uncommon to encounter errors in annotations, as seen in datasets like SemanticKITTI and Scannet, often due to factors like human error, sensor limitations, and varying conditions...

**Comment by haiphamcse** [comment]
Thank you for your quick reply, I think this kind of noise will affect the model's performance greatly. Have you tried to analyze how many times this occurs over the training set of SemanticKITTI? Furthermore what API did you use for DBScan, I'm currently using Open3D (don't know if there are any better alternatives)
FYI: It is very amazing that PaSCO can still output very decent panoptic predictions despite these type of noises  :D

**Comment by anhquancao** [comment]
Unfortunately, I can't recall the specific details as it's been about a year since I created this ground-truth data. However, I do remember training some methods with this data, and they produced quite accurate masks, so I didn't feel the need to make any adjustments.  
The results you've shared look really good. I had implemented DBSCAN on my own, but seeing your outputs, I think I should have followed your implementation.
